Core/Barcelona/Wrappers/Registry.swift

```diff
@@ -50,24 +46,8 @@ class Registry {
         IMAccountController.shared.accounts
     }
 
-    func imHandle(withID id: String, onService service: String) -> IMHandle? {
-        guard let account = bestAccount(for: service) else {
-            return nil
-        }
-
-        return imHandle(withID: id, onAccount: account)
-    }
-
-    func imHandle(withID id: String, onService service: IMService) -> IMHandle? {
-        guard let account = bestAccount(for: service) else {
-            return nil
-        }
-
-        return imHandle(withID: id, onAccount: account)
-    }
-
     func imHandle(withID id: String, onAccount account: IMAccount) -> IMHandle? {
-        account.imHandle(withID: id) ?? account.existingIMHandle(withID: id)
```

Comment:
Why did you remove the nil-coalescing stuff?

Reply:
According to the compiler it doesn't do anything.

Reply:
![20230406Ea1Skr8z](https://user-images.githubusercontent.com/9052157/230473215-bb104709-a4c6-4767-84ea-89884c8fca5b.png)

Reply:
> Left side of nil coalescing operator '??' has non-optional type 'IMHandle', so the right side is never used